### PR TITLE
Bugfix/#2844 create unit tests interceptor

### PR DESCRIPTION
**Pull request by AlekOK**
Unit tests

### CODE REVIEW

File: src/app/main/service/interceptors/interceptor.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+import { HttpRequest, HTTP_INTERCEPTORS } from '@angular/common/http';
+import { InterceptorService } from './interceptor.service';
+import { LocalStorageService } from '@global-service/localstorage/local-storage.service';
+import { UserOwnAuthService } from '@global-service/auth/user-own-auth.service';
+import { BehaviorSubject, of } from 'rxjs';
+import { MatSnackBarComponent } from '@global-errors/mat-snack-bar/mat-snack-bar.component';
+import { MatSnackBarModule } from '@angular/material';
+import { Router } from '@angular/router';
+
+describe(`InterceptorService`, () => {
```
Comment (JimHolman1199) [inline src/app/main/service/interceptors/interceptor.service.spec.ts:14]: it should be a single quote

File: src/app/main/service/interceptors/interceptor.service.spec.ts [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+  const mockAccessToken = 'testAccessToken';
+  const mockRefreshToken = 'testRefreshToken';
+  let router: Router;
+  let client: HttpClient;
```
Comment (JimHolman1199) [inline src/app/main/service/interceptors/interceptor.service.spec.ts:18]: do we need this ?

**Review by Edw1x — APPROVED** [review]

**Review by idzhersh — APPROVED** [review]

File: src/app/main/service/interceptors/interceptor.service.spec.ts (hunk 2 above)
Comment (AlekOK) [reply src/app/main/service/interceptors/interceptor.service.spec.ts:18]: No. I removed

**Review by rostikdoroshenko — APPROVED** [review]

**Comment by JimHolman1199** [comment]
Please fix single quote comment and open it again

**Pull request closed without merging**